source/backend/dal/Repositories/AcquisitionFilePropertyRepository.cs

```diff
@@ -62,6 +63,11 @@ public PimsPropertyAcquisitionFile Add(PimsPropertyAcquisitionFile propertyAcqui
         {
             propertyAcquisitionFile.ThrowIfNull(nameof(propertyAcquisitionFile));
 
+            if (propertyAcquisitionFile.Property.IsRetired.HasValue && propertyAcquisitionFile.Property.IsRetired.Value)
```

Comment:
I do sort of like having this logic at the repository level, because that means regardless of what service calls this, the user will be unable to add an acq file with a retired property. Is there a good argument for also having this at the service level?

Reply:
Not particularly besides short-circuiting the method and the fact that the repository Add is only called in this Service so it's more contained. But yeah, I guess it could be done in the repo.

Reply:
Updated